test/System/IntegrationTests/IntegrityTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using NUnit.Framework;
+
+namespace IntegrationTests
+{
+    [TestFixture]
+    class IntegrityTests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+        }
+
+        /// <summary>
+        /// This test scans "Dynamo/src" and sub-folders for "Resources.resx"
+        /// files, and tries to compare them with their "Resources.en-US.resx" 
+        /// counterparts to make sure they are exactly the same. The guideline 
+        /// is, everytime "Resources.resx" is updated, it should be copied to 
+        /// overwrite "Resources.en-US.resx" so that they are always in-sync.
+        /// </summary>
+        /// 
+        [Test]
+        public void AllResxFilesShouldMatchEnUsCounterparts()
+        {
+            var executingAsmPath = Assembly.GetExecutingAssembly().Location;
```

Comment:
I see, do we do that for all the time or under specific build configuration? If we do this all the time, then I may have to make the test succeed when `Dynamo/src` folder cannot be found.